packages/forms/src/model.ts

```diff
@@ -1450,22 +1450,36 @@ export class FormGroup extends AbstractControl {
    *
    * @param name The control name to add to the collection
    * @param control Provides the control for the given name
+   * @param opts Configuration options determine how the control propagates changes and emits
+   * events after the control is added.
```

Comment:
Since the `onlySelf` is not included into the options (which looks correct, if having `onlySelf` support would be needed in the future - it'd be a separate PR), we should update the docs to reflect that:
```suggestion
   * @param opts Allows to specify whether this FormGroup instance should emit events after a new control is added.
```
I believe docs for other functions should be rephrased a bit as well.